src/display/bitmaps.h

```diff
@@ -185,23 +185,19 @@ static const unsigned char Brew_Cup_Logo[] U8X8_PROGMEM = {
 };
 
 static const unsigned char Hot_Water_Logo[] U8X8_PROGMEM = {
-    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
-    0x18, 0x00, 0x00, 0x00, 0x00, 0x1c, 0x00, 0x00, 0x00, 0x00, 0x3c, 0x00, 
-    0x00, 0x00, 0x00, 0x3c, 0x00, 0x00, 0x00, 0x00, 0x7e, 0x00, 0x00, 0x00, 
-    0x00, 0xe7, 0x00, 0x00, 0x00, 0x00, 0xe7, 0x00, 0x00, 0x00, 0x00, 0xe7, 
-    0x00, 0x00, 0x00, 0xc0, 0xc3, 0x01, 0x00, 0x00, 0xc0, 0x81, 0x03, 0x00, 
-    0x00, 0xe0, 0x80, 0x07, 0x00, 0x00, 0x70, 0x00, 0x07, 0x00, 0x00, 0x70, 
-    0x00, 0x0e, 0x00, 0x00, 0x78, 0x00, 0x0e, 0x00, 0x00, 0x3c, 0x00, 0x1c, 
-    0x00, 0x00, 0x1c, 0x00, 0x38, 0x00, 0x00, 0x1c, 0x00, 0x38, 0x00, 0x00, 
-    0x1e, 0x00, 0x78, 0x00, 0x00, 0x0f, 0x00, 0x70, 0x00, 0x00, 0x07, 0x00, 
-    0x60, 0x00, 0x00, 0x07, 0x00, 0x60, 0x00, 0x00, 0x07, 0x00, 0x60, 0x00, 
-    0x00, 0x07, 0x00, 0x60, 0x00, 0x00, 0x07, 0x00, 0x60, 0x00, 0x00, 0x07, 
-    0x00, 0x60, 0x00, 0x00, 0x07, 0x00, 0x60, 0x00, 0x00, 0x07, 0x00, 0x60, 
-    0x00, 0x00, 0x0f, 0x00, 0x70, 0x00, 0x00, 0x1e, 0x00, 0x78, 0x00, 0x00, 
-    0x1c, 0x00, 0x38, 0x00, 0x00, 0x1c, 0x00, 0x18, 0x00, 0x00, 0x7c, 0x00, 
-    0x1f, 0x00, 0x00, 0xf0, 0x80, 0x0f, 0x00, 0x00, 0xf0, 0xff, 0x07, 0x00, 
-    0x00, 0x80, 0xff, 0x01, 0x00, 0x00, 0x00, 0xff, 0x00, 0x00, 0x00, 0x00, 
-    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+	0x00, 0x38, 0x00, 0x00, 0x00, 0x00, 0x38, 0x00, 0x00, 0x00, 0x00, 0x7c, 0x00, 0x18, 0x00, 0x00,
```

Comment:
Not that I care too much but the formatting of this block is now different than in the rest of the file 😄

Reply:
I didn't liked it also, but was too late for fidily around with it. :D